content/en/docs/demo/kubernetes-deployment.md

```diff
@@ -121,16 +121,16 @@ the proper support before using these configuration options.
 
 {{% /alert %}}
 
-Each demo component (ie: frontendproxy) offers a way to have its Kubernetes
+Each demo component (ie: frontend-proxy) offers a way to have its Kubernetes
 service type configured. By default, these will not be created, but you can
 enable and configure them through the `ingress` property of each component.
 
-To configure the frontendproxy component to use an ingress resource you would
+To configure the frontend-proxy component to use an ingress resource you would
 specify the following in your values file:
```

Comment:
I think that this prose (and the similar prose in the next section) needs to be reworked in a followup PR. Just taking note for now.

Reply:
Will track via:

- #6465